README.md

```diff
@@ -85,9 +85,9 @@ A tool with similar goals but a different implementation is
 [codespell](https://pypi.org/project/codespell/).
 
 The main advantage of codespell is that it seems to have
-many fewer false positives.
+many fewer false positives as it uses its own dictionaries
+of typos.
 
-The main advantage of kodespel is that it checks identifiers,
-not just comments and strings,
```

Comment:
FWIW, codespell doesn't know anything about the syntax of a file (for better or worse).